<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AC_Control</class>
 <widget class="QMainWindow" name="AC_Control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>750</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AC_Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>671</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS UI Gothic</family>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;test&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Nullpunkt_controltab">
     <attribute name="title">
      <string>Standa</string>
     </attribute>
     <widget class="QLabel" name="Position">
      <property name="geometry">
       <rect>
        <x>196</x>
        <y>112</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Position</string>
      </property>
     </widget>
     <widget class="QLabel" name="uSteps">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>uSteps</string>
      </property>
     </widget>
     <widget class="QToolButton" name="Go_left_Button">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>210</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
     <widget class="QToolButton" name="Go_Home_Button">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>350</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Go Home</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Pos_spinBox">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="move_to_Button">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>330</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>move to</string>
      </property>
     </widget>
     <widget class="QToolButton" name="Go_right_Button">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="Stop_Button">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QLabel" name="Steps">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Steps</string>
      </property>
     </widget>
     <widget class="QToolButton" name="SetHome_Button">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>370</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Home</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="uPos_spinBox">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>290</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="uPosition">
      <property name="geometry">
       <rect>
        <x>186</x>
        <y>170</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>uPosition</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Standa_Connect_Button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Connect to Standa</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>connected</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Standa_Close_Button">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Close connection to Standa</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="standa_connection_ip">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="standa_label_ip">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="standa_label_port">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="standa_connection_port">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pyrpltab">
     <attribute name="title">
      <string>Pyrpl</string>
     </attribute>
     <widget class="QPushButton" name="Pyrpl_Connect_Button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Connect to Pyrpl</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Pyrpl_Close_Button">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Close connection to Pyrpl</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxPyrpl_Connect_CheckBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Connected to Pyrpl</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pyrpl_connection_port">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pyrpl_label_port">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="pyrpl_label_ip">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Messung">
     <property name="font">
      <font>
       <pointsize>4</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>Messung</string>
     </attribute>
     <widget class="QPushButton" name="Add_to_list_Button">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add to List</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Start_SpinBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="End_SpinBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Stepsize_SpinBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="StartPoint">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start Point</string>
      </property>
     </widget>
     <widget class="QLabel" name="EndPoint">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>End Point</string>
      </property>
     </widget>
     <widget class="QLabel" name="StepSize">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Step Size</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>400</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="filePath_Edit">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>380</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>choose file</string>
      </property>
     </widget>
     <widget class="QToolButton" name="select_file_Button">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>380</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wählen....</string>
      </property>
     </widget>
     <widget class="QLabel" name="list_label">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>List:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>380</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dateipfad</string>
      </property>
     </widget>
     <widget class="QListWidget" name="step_list">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>81</width>
        <height>192</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="Clear_list_Button">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clear List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Load_list_Button">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Save_list_Button">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save List</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ac_checkBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>360</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>AC1 mit Loggen?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Sort_list_Button">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>150</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sort List</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>570</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>print()</string>
    </property>
   </widget>
   <widget class="QListWidget" name="print_list">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>550</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
